src/test/java/com/akademia/projectplanner/validation/TaskValidatorTest.java

```diff
@@ -0,0 +1,67 @@
+package com.akademia.projectplanner.validation;
+
+import com.akademia.projectplanner.dto.TaskDto;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TaskValidatorTest {
+
+  private TaskDto taskDto;
+
+  @BeforeEach
+  void beforeEach() {
+    taskDto = new TaskDto();
+  }
+
+  @Test
```

Comment:
Trochę poprawek stylistyczno-konwencyjnych, dotyczy całości:
- z tymi nazwami to nie powinniśmy powielać słowa "test" bo wiemy że to test, raczej stosuje się coś w stylu: shouldPassWith.. , throwExceptionWhenCośTam, ifArgumentIsCośTamToCośTam
- trochę opisowo ale nazwy testów czasami mogą być czytane przez devopsów, klientów itp w logach więc muszą być znaczące

  @Test
  void shouldBeFalseWithCorrectStatusAndName() {
    //given
    taskDto.setStatus("In progress");
    taskDto.setName("Task number 1");
    //when
    boolean result = TaskValidator.hasBlankNameOrStatus(taskDto);
    //then
    assertFalse(result);
  }